package.json

```diff
@@ -37,6 +37,7 @@
     "expo": "^41.0.0",
     "expo-asset": "~8.3.1",
     "expo-cli": "^4.4.8",
+    "expo-clipboard": "^1.0.2",
```

Comment:
This is the only supported clipboard package for Expo. Native `Clipboard` is deprecated and the suggested community package is not supported by Expo